src/server/pfs/server/driver.go

```diff
@@ -523,7 +523,7 @@ func (d *driver) startCommit(
 			return nil, errors.Wrapf(err, "parent commit not found")
 		}
 		// fail if the parent commit has not been finished
-		if parentCommitInfo.Finished == nil {
+		if parentCommitInfo.Finishing == nil {
```

Comment:
Why did this need to change?  Also the error message is not in sync.

Reply:
If you call finish commit, then start quickly, it would error if we did not allow you to start a commit until the parent was finished (as compared to finishing). Any reads on the commit would block until the parent is finished. I was thinking about keeping the error as finished commit for both finishing / finished in the interest of time.